### PR TITLE
Add copy for /credentialling/self-study

### PR DESCRIPTION
**Pull request by mrgnr**
## Done

- Add copy for /credentialling/self-study

## QA

- Visit the [demo](https://ubuntu-com-12066.demos.haus/credentialling/self-study)
  - Verify that the copy matches the [copy doc](https://docs.google.com/document/d/1H_ug8HenT_qVXGachnj0bzf3MWSrUFc83scAmwtsyNo)

## Issue / Card

Fixes https://github.com/canonical/commercial-squad/issues/720
Fixes https://warthogs.atlassian.net/browse/CRED-162

## Screenshots

cred-self-study

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/credentialling/self-study.html [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+      <p>There are a number of instructional resources available, both community- and Canonical-made, that contain a wealth of useful information. The following links lead to Canonical’s official educational resources.</p>
+      <ul>
+        <li>
+          <a href="https://ubuntu.com/tutorials">Tutorials</a>
+        </li>
+        <li>
+          <a href="https://help.ubuntu.com/">Ubuntu documentation</a>
+        </li>
+        <li>
+          <ul>
```
Comment (lex-joy) [inline templates/credentialling/self-study.html:79]: Something is going wrong here. We introduce two bullets in front of the Desktop entry somehow. We need to eliminate the first, and have only the nested bullets.

Once that's resolved, this is perfect.

Comment (lex-joy) [reply templates/credentialling/self-study.html:79]: Screenshot from 2022-09-30 13-11-32

Comment (mrgnr) [reply templates/credentialling/self-study.html:77]: Good catch, I've made the fix.

**Comment by lex-joy** [comment]
Looks terrific! Thank you.

**Review by lex-joy — APPROVED** [review]

**Review by abhigyanghosh30 — APPROVED** [review]

**Pull request merged**